tests/models/test_keg.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+from unittest import mock
+
+from pyplaato.models.keg import PlaatoKeg
+
+
+def test_date_prop_with_value_returns_its_timestamp():
+    pins = PlaatoKeg.Pins
+    keg = PlaatoKeg({pins.DATE: "1/1/2022"})
+    assert 1641024000.0 == keg.date
```

Comment:
This test verifies it's returning the correct timestamp and not the current one.